src/js/components/Page.js

```diff
@@ -179,6 +180,23 @@ var Page = React.createClass({
     );
   },
 
+  getLicensingBanner() {
+    // todo use mountservice and mount LicenseBanner
+    // which will render the Banner as a child
+    // return <MountService.Mount type="Page:LicenseBanner" />
+
+    // for testing, hardcode return the banner component
+    return (
+      <Banner>
+        You have exceeded your 60 day limit by 2 days.
+        {" "}
+        <a href="https://www.paypal.me/arhoskins" className="message-link">
```

Comment:
request changes `https://www.paypal.me/weblancaster`

Reply:
😂

Reply:
We aren't merging it like this are we? :) Suggestion: replace the text and the link with something neutral, that doesn't have licensing traces, i.e. https://mesosphere.com/product/

Reply:
Lol, I've changed this to mesosphere.com/product for now. We should eventually have a specific docs page we'll direct to, but that doesn't exist yet.